manifests/d/DuckStudio/GitHubLabelsManager/v1.1/DuckStudio.GitHubLabelsManager.installer.yaml

```diff
@@ -0,0 +1,43 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: "v1.1"
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent /suppressmsgboxes
+  SilentWithProgress: /verysilent
+UpgradeBehavior: install
+Commands:
+- glm
+Protocols:
+- http
+- https
+ReleaseDate: 2024-07-04
+AppsAndFeaturesEntries:
+- DisplayVersion: 'v1.1'
```

Comment:
```suggestion
```

Reply:
I made an error in this suggestion - it should remove both the `DisplayVersion` and also the `AppsAndFeaturesEntries` line above.

Reply:
Done.